content/geschaftsordnung.md

```diff
@@ -11,122 +11,108 @@ tocopen: true
 > Eine PDF Version kann unter [https://brokkoli-freunde.de/geschaeftsordnung.pdf](/geschaeftsordnung.pdf) eingesehen werden.
 <!--more-->
 
-## 1 Geltungsbereich
+## § 1 Geltungsbereich
 Diese Geschäftsordnung gilt für die Mitgliederversammlung (im folgenden Festessen) der
 Brokkoli Freunde e.F., soweit die Satzung des Brokkoli Freunde e.F. nicht eine andere
 Regelung trifft.
 
-## 2 Termin
+## § 2 Termin
 Ein Festessen wird vom Vorstand einberufen. Der Termin des Festessens wird vom Vorstand
 in Abstimmung mit den Vereinsmitgliedern beschlossen. Eine außerplanmäßiges Festessen
 muss einberufen werden, wenn mindestens die Hälfte der Vereinsmitglieder dies wünscht.
 
-## 3 Vorbereitung
+## § 3 Vorbereitung
 Die Vorbereitung des Festessens erfolgt durch den Vorstand oder wird vom Vorstand delegiert.
 
-## 4 Vorläufige Tagesordnung (im folgenden Menü)
+## § 4 Vorläufige Tagesordnung (im folgenden Menü)
 Das vorläufige Menü des Festessens wird im Vorstand beraten und beschlossen.
 
-## 5 Unterlagen
+## § 5 Unterlagen
 Rechtzeitig vor dem Festessen erhalten die Mitglieder der Brokkoli Freunde e.F. durch den
 Vorstand die notwendigen Unterlagen. Diese bestehen aus der Einladung, die Ort, Datum und
 Menü des bevorstehenden Festessens enthalten muss und aus den eingegangenen Anträgen.
 
-## 6 Beginn der Sitzung
+## § 6 Beginn der Sitzung
 Das Festessen beginnt mit Feststellung der Anwesenheit und der Beschlussfähigkeit und
 Festlegung des endgültigen Menüs.
 
-## 7 Menü
+## § 7 Menü
 Weitere Tagesordnungspunkte (im folgenden Gänge) können von jedem Vereinsmitglied in
 das Menü eingebracht werden. Zu jedem Gang können während des Festessens Anträge
 gestellt werden, die schriftlich festzuhalten sind und über die nach der einfachen Mehrheit der
 anwesenden Stimmberechtigten Vereinsmitglieder abgestimmt wird.
 
-## 8 Schluss des Festessens
+## § 8 Schluss des Festessens
 Das Festessen endet nach Abarbeitung des Menüs. Der Beschluss zum vorzeitigen Beenden
 des Festessens bedarf einer Zwei-Drittel-Mehrheit der anwesenden stimmberechtigten
 Mitglieder.
 
-## 9 Beschlussfähigkeit
+## § 9 Beschlussfähigkeit
 Das Festessen ist beschlussfähig, wenn ordnungsgemäß eingeladen wurde.
 Zusätzlich ist das Vorhandensein oder das glaubhaft in Aussicht gestellte Vorhandensein von
 Brokkoli oder eines Brokkoli-Äquivalents während des Festessens notwendig. Ein Brokkoli-
 Äquivalent muss mit Zwei-Drittel-Mehrheit als solches akzeptiert werden. Das Vorhandensein
 einschließlich der Akzeptanz darf zu einem beliebigen Zeitpunkt des Festessens erfolgen und
 somit auch das bisherige Festessen legitimieren. Ausgeschlossen von dieser Äquivalenz-
 Akzeptanz sind Objekte nach Anlage I-III zu ##1 Absatz 1 des BtMG. Konsum dieser führt zu
```

Comment:
up for discussion

Reply:
Either would be fine for me, your version is probably better

Reply:
Either would be fine for me, your version is probably better

Reply:
right now there is only one option presented (line 56 f)